modules/network/pre-existing-subnetwork/README.md

```diff
@@ -0,0 +1,97 @@
+## Description
+
+This module is inspired by pre-existing-vpc, but is changed with the objective of supporting Shared VPCs.
+
+For example, the blueprint below discovers the referred to subnetwork. 
+With the `use` keyword, the [vm-instance] module accepts the `subnetwork_self_link`
+input variables that uniquely identify the subnetwork in which the VM will be created.
+
+[vpc]: ../vpc/README.md
+[vm-instance]: ../../compute/vm-instance/README.md
+
+> **_NOTE:_** Additional IAM work is needed for this to work correctly.
+
+### Example
+
+```yaml
+- id: network1
+  source: modules/network/pre-existing-subnetwork
+  settings:
+    subnetwork_name: name-of-subnetwork
```

Comment:
AFAIK supplying `subnetwork_name ` along `subnetwork_self_link`  is meaningless, correct? 
Example should showcase proper usage, not list of all fields.